client/index.html

```diff
@@ -33,12 +35,21 @@
         thead.appendChild(header);
         tableData[0]['values'].forEach(row => {
             var tr = document.createElement('tr');
-            row.forEach(cell => {
+            row.forEach((cell, index) => {
+                if(index==0){ // delete first column "id"
+                    row.deleteCell(0);
```

Comment:
My browser shows that execution is stopping here. Because this code is a function called for each cell, I think you could

```suggestion
                    return;
```

and it would work as intended, without creating a cell. The code above that creates a header would also need to be modified.